### PR TITLE
data:Taheer10

### PR DESCRIPTION
**Pull request by Taheer10**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4508"></a>

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
you created your json file in the leading directory. You need to make it into `data` directory `data/your-GitHub-user-name.json`

Just follow this video guide https://www.youtube.com/watch?v=05HEeCQSKRE&t=2s

File: Taheer10.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Tahir Hussain",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learn to",
+  "avatar":"https://github.com/Taheer10.png"
```
[the remark this answers is not in the document]
Comment (Taheer10) [reply Taheer10.json:6]: How do i update  bio and change directory. The video does not show anything regarding update.

File: data/Taheer10.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Tahir Hussain",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a frontend Developerand Love to Learn New technologies",
```
Comment (priyankarpal) [inline data/Taheer10.json:5]:
``` suggestion
  "bio": "I am a frontend Developer and Love to Learn New technologies",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by Taheer10 — COMMENTED** [review]
Reviwed.

**Comment by priyankarpal** [comment]
> Reviwed.

Just fix this small problem

**Review by Taheer10 — COMMENTED** [review]
Done

**Comment by Taheer10** [comment]
What do i do now ?

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Comment by priyankarpal** [comment]
> What do i do now ?

Just wait It will merge soon by the maintainer.

**Review by Sindhuinti — APPROVED** [review]
LGTM 👍🏻

**Review by Hrishi5111998 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**